airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -265,7 +265,7 @@
                 "type": "integer",
                 "title": "Initial Waiting Time in Seconds (Advanced)",
                 "description": "The amount of time the connector will wait when it launches to determine if there is new data to sync or not. Defaults to 300 seconds. Valid range: 120 seconds to 1200 seconds. Read about <a href=\"https://docs.airbyte.com/integrations/sources/postgres#step-5-optional-set-up-initial-waiting-time\">initial waiting time</a>.",
```

Comment:
Update the description, it says it defaults to 300 seconds. Also, consider raising the upper bound of the valid range, perhaps?

Reply:
Good catch - done!